packages/conf-magick-core6/conf-magick-core6.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "https://github.com/fccm2"
+homepage: "https://imagemagick.org/script/magick-core.php"
+license: "https://imagemagick.org/script/license.php"
```

Comment:
To resolve the linting error 

```
Error in conf-magick-core7.1: warning 62: License doesn't adhere to the SPDX standard, see https://spdx.org/licenses/ : "https://imagemagick.org/script/license.php"
```

I think this should be


```suggestion
license: "ImageMagick"
```

as per https://spdx.org/licenses/ImageMagick.html